app/decorators/staff/time_slots_decorator.rb

```diff
@@ -1,4 +1,4 @@
-class TimeSlotsDecorator < Draper::CollectionDecorator
+class Staff::TimeSlotsDecorator < Draper::CollectionDecorator
```

Comment:
I looked at how the decorator was being used, and it was decidedly staff-only. Since Draper apparently check directories recursively, it made all the uses of this decorator more verbose. I've got a TODO to look into configuration options to help clean that up, but didn't bother on my first pass.